### PR TITLE
Fix Camera permissions for signed Apps

### PR DESCRIPTION
**Pull request by awlx**
This fixes the camera permissions if the app needs to be signed.

Should also fix:
#299

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,9 @@
       "artifactName": "jitsi-meet.${ext}",
       "category": "public.app-category.video",
       "darkModeSupport": true,
-      "hardenedRuntime": false,
+      "hardenedRuntime": true,
```
Comment (saghul) [inline package.json:32]: Does this also work in the non-signed case?

Comment (awlx) [reply package.json:32]: It should not, as the entitlements also get used to ask for permission.

Comment (awlx) [reply package.json:32]: https://developer.apple.com/documentation/security/hardened_runtime?preferredLanguage=occ

**Comment by saghul** [comment]
Verified it works on unsigned apps too, thanks a lot @awlx !

**Pull request merged**